boards/limifrog-v1/include/periph_conf.h

```diff
@@ -27,13 +27,13 @@ extern "C" {
  * @name Clock system configuration
  * @{
  **/
-#define CLOCK_HSE           (16000000U)         /* external oscillator */
+#define CLOCK_HSI           (16000000U)         /* external oscillator */
```

Comment:
I was referring to the comment `external oscillator` which it isn't anymore, right?